test_app/tests/tests_views/tests_nested_views.py

```diff
@@ -26,11 +26,9 @@ def test_list_endpoint(self):
         self.assertEqual(2, len(data['results']))
 
         expected_data = [
-            self.expected_detailed_wizards[1],
-            self.expected_detailed_wizards[0],
+            self.expected_wizards[1],
+            self.expected_wizards[0],
```

Comment:
will this break something that is using something that is not available anymore?

Reply:
It might.
The apps that use this package are responsible for their tests, and the `drf-kit` allows customization to reproduce the previous behavior if these apps do not want to use this fix.

Reply:
The problem is that the ones using the API will be the ones broken and not the API itself. I think the ones migrated should have the deploy checking for possible integration problems.